services/movie.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet | Movie:

```

Comment:
Return type hint here will be `QuerySet` 

Remember:`.all()`, `.filter()` returns QuerySet`
.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example